config.js

```diff
@@ -556,6 +556,7 @@ assert(config.NAMESPACE_CACHING.DEFAULT_BLOCK_SIZE > config.INLINE_MAX_SIZE);
 
 config.NSFS_BUF_SIZE = 8 * 1024 * 1024;
 config.NSFS_BUF_POOL_MEM_LIMIT = config.BUFFERS_MEM_LIMIT;
+config.NSFS_IOV_MAX = 1024;
```

Comment:
I thought maybe we should read it from the target platform headers, but it's also fine as a config I guess.
I would at least add a comment to reference this to the actual limit -
```suggestion
config.NSFS_IOV_MAX = 1024; // see IOV_MAX in https://man7.org/linux/man-pages/man0/limits.h.0p.html
```

Reply:
I agree, I didn't find an easy way to do it in node js so I read it in the native layer and kept the default in config, let me know what you think

Reply:
I added the IOV_MAX value reading from the platform and the comment